library/ix-dev/community/netbootxyz/Chart.yaml

```
@@ -0,0 +1,30 @@
+name: netbootxyz
+description: netboot.xyz lets you PXE boot various operating system installers or utilities from a single tool over the network.
+annotations:
+  title: netbootxyz
+type: application
+version: 1.0.0
+apiVersion: v2
+appVersion: 2.0.76
+kubeVersion: '>=1.16.0-0'
+maintainers:
+  - name: truenas
+    url: https://www.truenas.com/
+    email: dev@ixsystems.com
+dependencies:
+  - name: common
+    repository: file://../../../common
+    version: 1.2.7
```

```suggestion
    version: 1.2.9
```

Pull the latest master and run `helm dep update`!